_posts/en/pages/2016-01-15-lifecycle.md

```diff
@@ -53,9 +53,23 @@ Once EOL is reached, you will need to upgrade to a newer version.
 
 _TBA: to be announced_
 
-## Foot notes
+## Protocol versioning
```

Comment:
This talks about the literal numbers, but doesn't mention the actual consensus protocol itself, which (for better or worse) is no longer indicated by any given number.